cors/cors-safelisted-request-header.any.js

```diff
@@ -40,4 +40,18 @@ function safelist(headers, expectPreflight = false) {
   ["multipart/form-data;\"", true]
 ].forEach(([mimeType, preflight = false]) => {
   safelist({"content-type": mimeType}, preflight);
-})
+});
+
+[
+  ["100-200", true],
+  ["MB=100-200", true],
+  ["bytes=100-200"],
+  ["BYTES=100-200"],
```

Comment:
The spec doesn't currently allow this. Is it case insensitive in the HTTP spec?

Reply:
I thought headers were generally case-insensitive. I removed the test case.